src/main/plugin/iso19139.ca.HNAP/layout/utility-tpl-multilingual.xsl

```diff
@@ -47,7 +48,19 @@
             <xsl:value-of select="$metadata/gmd:language/gmd:LanguageCode/@codeListValue"/>
           </xsl:when>
           <xsl:when test="contains($metadata/gmd:language/gco:CharacterString,';')">
-            <xsl:value-of  select="normalize-space(substring-before($metadata/gmd:language/gco:CharacterString,';'))"/>
+            <xsl:variable name="metadataMainLanguage">
+              <xsl:call-template name="langId_from_gmdlanguage19139">
+                <xsl:with-param name="gmdlanguage" select="/root/*/gmd:language"/>
+              </xsl:call-template>
+            </xsl:variable>
+            <xsl:choose>
```

Comment:
This choose clause can probably call a function because it's been used multiple times within this code change.